router/pkg/config/testdata/config_full.json

```diff
@@ -217,17 +217,42 @@
   },
   "DevelopmentMode": false,
   "Events": {
-    "Sources": {
-      "another-nats": {
-        "Provider": "NATS",
-        "URL": "nats://localhost:4223",
-        "Authentication": null
-      },
-      "default": {
-        "Provider": "NATS",
-        "URL": "nats://localhost:4222",
-        "Authentication": null
-      }
+    "Providers": {
+      "Nats": [
+        {
+          "ID": "default",
```

Comment:
The default nats provider id is now "nats" rather than "default".

Just making sure that "default" here is intended.

Reply:
Done